packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -1004,10 +835,35 @@ abstract class ResidentRunner {
 
   /// Called when a signal has requested we exit.
   Future<void> cleanupAfterSignal();
+
   /// Called right before we exit.
   Future<void> cleanupAtFinish();
+
   /// Called when the runner should handle a terminal command.
-  Future<void> handleTerminalCommand(String code);
+  Future<void> handleTerminalCommand(String code) async {
+    final String lower = code.toLowerCase();
```

Comment:
Can we turn this into the switch case statement?  I think it will make reading this easier.

Reply:
Done